spiffworkflow-backend/src/spiffworkflow_backend/services/process_instance_service.py

```diff
@@ -209,6 +218,97 @@ def get_users_assigned_to_task(
 
             return lane_uids
 
+    @classmethod
+    def file_data_model_for_value(
+        cls,
+        identifier: str,
+        value: str,
+        process_instance_id: int,
+    ) -> Optional[ProcessInstanceFileDataModel]:
+        if value.startswith("data:"):
```

Comment:
i wonder if this would be better as "spiffworkflowdata:". not sure how likely either are to come up in actual use.

Reply:
That one has to stay as "data:" since it is the `data-url` format that the frontend requires. Once its uploaded, inside the `base64,` part, we prefix our hash with `spifffiledatadigest+`.